### ISSUE TITLE
Returned archive info has changed breaking updating of Block status

### ISSUE THREAD

**Issue by talister** [LCOGT/neoexchange#601]
```
INFO:core:Request no. 2772195 x 4 images (4 total all red. levels)
Traceback (most recent call last):
  File "/home/tlister/git/neoexchange-devel/neoexchange/manage.py", line 42, in <module>
    execute_from_command_line(sys.argv)
  File "/home/tlister/venv/neoexchange39_venv/lib64/python3.9/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/home/tlister/venv/neoexchange39_venv/lib64/python3.9/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/tlister/venv/neoexchange39_venv/lib64/python3.9/site-packages/django/core/management/base.py", line 330, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/tlister/venv/neoexchange39_venv/lib64/python3.9/site-packages/django/core/management/base.py", line 371, in execute
    output = self.handle(*args, **options)
  File "/home/tlister/git/neoexchange-devel/neoexchange/core/management/commands/update_blocks.py", line 32, in handle
    block_status(block.id)
  File "/home/tlister/git/neoexchange-devel/neoexchange/core/frames.py", line 380, in block_status
    last_image = datetime.strptime(last_image_header['data']['DATE_OBS'][:19], '%Y-%m-%dT%H:%M:%S')
KeyError: 'DATE_OBS'
```

**Comment by talister** [comment]
So the return from `lco_api_call()` now has `DATE-OBS` rather than `DATE_OBS` as it has forever... which is why it's breaking

**Comment by talister** [comment]
Fixed in `issue-601` and PR #602
